src/include/snapper/dialogs.rb

```diff
@@ -465,11 +473,7 @@ def SummaryDialog
         Popup.ClearFeedback
 
         UI.ChangeWidget(Id(:snapshots_table), :Items, get_snapshot_items.call)
-        UI.ChangeWidget(
-          Id(:show),
-          :Enabled,
-          Ops.greater_than(Builtins.size(snapshot_items), 0)
-        )
+	enable_snapshot_buttons(Ops.greater_than(Builtins.size(snapshot_items), 0))
```

Comment:
Please, take the chance to remove autotranslated code.
```Ops.greater_than(Builtins.size(snapshot_items), 0)``` looks nicer as ```!snapshots_items.empty?``` or something similar.